src/routes/(authentication)/login/loading/+page.svelte

```diff
@@ -60,15 +59,6 @@
       goto('/login/error');
       return;
     });
-
-    let filtersJson = sessionStorage.getItem('filters');
-    if (filtersJson) {
-      let storedFilters = JSON.parse(filtersJson || '[]');
-      filters.set(storedFilters);
-      // wait to delete from session storage, in case loading the filters in the line above triggers the session
-      // storage to be re-written
-      setTimeout(() => sessionStorage.setItem('filters', '[]'), 500);
-    }
```

Comment:
I think this was here to prevent users from having "bad" filters the didnt have access to in BDC and they would get stuck in a log out loop. @ramari16 might remember.